src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -46,16 +46,16 @@ export function initiateMedicationDispenseBody(
       medicationReference: medicationRequest.medicationReference,
       medicationCodeableConcept: medicationRequest.medicationCodeableConcept,
       subject: medicationRequest.subject,
-      /* performer: [
+      performer: [
         {
           actor: {
-            reference: medicationRequest.requester.reference,
+            reference: "Practitioner/" + session?.currentProvider?.uuid,
```

Comment:
It would probably be better to write:

```suggestion
            reference: session?.currentProvider ? `Practitioner/${session.currentProvider.uuid}` ; ""
```

Reply:
Good point, and agreed @ibacher !  I have updated.